Dockerfile-e2e-runner

```diff
@@ -0,0 +1,13 @@
+FROM mhart/alpine-node:5.11.1
```

Comment:
this alpine image is 95 MB compared to the main node image which is 749 MB. should consider using in another branch where change is more isolated